### ISSUE TITLE
Newbie question: why are scalar exif properties (like DateTimeOriginal and most others) returned as string arrays?

### ISSUE THREAD

**Issue by ehahn9** [mattiasw/ExifReader#272]
For example:

```
    'ExifVersion'?: NumberArrayTag,
    'DateTimeOriginal'?: StringArrayTag,
    'DateTimeDigitized'?: StringArrayTag,
```

As a developer, I was surprised by this, especially since many other values (like `ExposureTime`) are scalars.

I'm sure I'm missing something! Perhaps it is always safe to use `value[0]` for such arrays? Thanks for any advice you have on this. It is a *fantastic* library!!

**Comment by mattiasw** [comment]
Hi! The `value` tries to represent how the tags are stored in the image. It's mostly better to just use `description` unless you really want to make sure the value has the exact format you need, and in that case you may need to dig into the Exif specification to know how to use it, depending on which tag we're talking about.

That `ExposureTime` (and others) is not an array is probably a bug in the types. Its value is stored as two numbers (numerator and denominator) and should therefore be an array of two numbers. To get the real value for that it's therefore not correct to just use `value[0]`.

I'm less sure about why e.g. `DateTimeOriginal` is an array of strings (here the type correctly reflects the real value). It shouldn't have to be unless I'm missing something (some tags are repeatable but not this one). Maybe for consistency (hard to remember after 10+ years 😅).

Short answer: try to use `description` where possible.

**Comment by ehahn9** [comment]
thx. I very much appreciate your time on the reply and of course on the library itself!

**Comment by mattiasw** [comment]
:+1: I will take a look at the suspicious types when I have time. Thanks for bringing it to my attention.

**Comment by mattiasw** [comment]
Several faulty types, `ExposureTime` and others, were fixed in version 4.23.6.
